app/models/course/condition/assessment.rb

```diff
@@ -0,0 +1,36 @@
+class Course::Condition::Assessment < ActiveRecord::Base
+  include ActiveSupport::NumberHelper
+
+  acts_as_condition
+  belongs_to :assessment, class_name: Course::Assessment.name, inverse_of: false
+
+  default_scope { includes(:assessment) }
+
+  def title
+    if minimum_grade_percentage
+      self.class.human_attribute_name(title.title,
```

Comment:
This prints the assessment condition using I18n. The output should be "Mission 2 (at least 33.33%)". It was 2 lines originally but the coverage failed and I couldn't understand why.